src/pod.ts

```diff
@@ -111,6 +111,8 @@ export class Pod {
           require('sucrase/register/ts');
         }
         const configFilename = this.getAbsoluteFilePath(podPath);
+        // Allow runtime reloading of the config file.
+        delete require.cache[require.resolve(configFilename)];
```

Comment:
can you confirm this doesn't error for projects that don't have a amagaki.js file? or is the file always required for every proj?

Reply:
On line 109 it checks to see that the file exists beforehand, so this should be ok on the file check part.

Reply:
Yea, it works without it. We have a check further up in the code block -- this only runs if the file exists.